src/navigation-instruction.ts

```diff
@@ -204,7 +217,7 @@ export class NavigationInstruction {
     let delaySwaps = [];
 
     for (let viewPortName in this.viewPortInstructions) {
-      let viewPortInstruction = this.viewPortInstructions[viewPortName];
+      let viewPortInstruction: ViewPortInstruction = this.viewPortInstructions[viewPortName];
```

Comment:
unnecessary

Reply:
Got the typing in before `this.viewPortInstructions`, Removing next commit